.github/actions/yarn-install/action.yml

```
@@ -4,16 +4,32 @@ inputs:
     type: string
     required: false
     default: '20.x'
+  recreate-yarn-lock:
```

nit: Maybe simplify name, this isn't recreated as an _input_. What are we doing with the recreated file outputs if there's a diff?
```suggestion
  no-yarn-lock:
```